src/main/java/org/mitre/synthea/export/Exporter.java

```diff
@@ -418,52 +432,21 @@ private static boolean entryWithinTimeRange(
   }
 
   /**
-   * Get the folder where the patient record should be stored.
-   * See the configuration settings "exporter.subfolders_by_id_substring" and
-   * "exporter.baseDirectory".
-   *
-   * @param folderName The base folder to use.
-   * @param person     The person being exported.
-   * @return Either the base folder provided, or a subdirectory, depending on configuration
-   *     settings.
-   */
-  public static File getOutputFolder(String folderName, Person person) {
-    List<String> folders = new ArrayList<>();
-
-    folders.add(folderName);
-
-    if (person != null
-        && Boolean.parseBoolean(Config.get("exporter.subfolders_by_id_substring"))) {
-      String id = (String) person.attributes.get(Person.ID);
-
-      folders.add(id.substring(0, 2));
-      folders.add(id.substring(0, 3));
-    }
-
-    String baseDirectory = Config.get("exporter.baseDirectory");
-
-    File f = Paths.get(baseDirectory, folders.toArray(new String[0])).toFile();
-    f.mkdirs();
-
-    return f;
-  }
-
-  /**
-   * Get the filename to used to export the patient record.
-   * See the configuration setting "exporter.use_uuid_filenames".
+   * Get the fileName to used to export the patient record. See the configuration
+   * setting "exporter.use_uuid_fileNames".
    *
    * @param person    The person being exported.
-   * @param tag       A tag to add to the filename before the extension.
+   * @param tag       A tag to add to the fileName before the extension.
    * @param extension The file extension to use.
-   * @return The filename only (not a path).
+   * @return The fileName only (not a path).
    */
   public static String filename(Person person, String tag, String extension) {
-    if (Boolean.parseBoolean(Config.get("exporter.use_uuid_filenames"))) {
+    if (Boolean.parseBoolean(Config.get("exporter.use_uuid_fileNames"))) {
```

Comment:
You changed the configuration setting name without changing the configuration file.

The configuration settings do not use camel-case, please revert to `exporter.use_uuid_filenames`

Reply:
fixed